scanpullrequest/scanpullrequest.go

```diff
@@ -100,9 +99,9 @@ func scanPullRequest(repo *utils.Repository, client vcsclient.VcsClient) (err er
 	}
 
 	// Output results
-	shouldSendExposedSecretsEmail := issues.SecretsExists() && repo.SmtpServer != ""
+	shouldSendExposedSecretsEmail := issues.SecretsIssuesExists() && repo.SmtpServer != ""
 	if shouldSendExposedSecretsEmail {
-		secretsEmailDetails := utils.NewSecretsEmailDetails(client, repo, issues.Secrets)
+		secretsEmailDetails := utils.NewSecretsEmailDetails(client, repo, append(issues.SecretsVulnerabilities, issues.SecretsViolations...))
```

Comment:
would we need the secrets violations and vulnerabilities seperated at later stage?
Just for the sake we will we might want to combine them in a different var and not in the original resutls struct